### PR TITLE
Add moth plushie

### PR DESCRIPTION
**Pull request by Fromoriss**
About the PR
This will add moth plushie in plushie crate.

 Why / Balance
I forget it yesterday..

 Technical details
Only makes plushie crate (can be ordered in cargo) filled with moth plushie.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
No cl

### CODE REVIEW

**Comment by Fromoriss** [comment]
Ok, somebody already makes it, with arcades..

**Comment by deltanedas** [comment]
reopen that pr doesnt add the plushie itself

**Comment by Fromoriss** [comment]
> reopen that pr doesnt add the plushie itself

There is another PR, that add plushie both to crate and arcade.

**Comment by deltanedas** [comment]
yes but it depends on this

**Comment by Partmedia** [comment]
Could you please clarify the license for this moth plushie sprite, or where you got it from?

**Comment by LankLTE** [comment]
> Could you please clarify the license for this moth plushie sprite, or where you got it from?

It’s taken from /tg/, I had assumed the commit at the top of the toys.rsi meta would cover it

**Comment by Partmedia** [comment]
> > Could you please clarify the license for this moth plushie sprite, or where you got it from?
> 
> It’s taken from /tg/, I had assumed the commit at the top of the toys.rsi meta would cover it

It does, I wanted to double check that this is actually from the same source.

**Review by lzk228 — CHANGES_REQUESTED** [review]
merge master into your branch

**Comment by Partmedia** [comment]
> merge master into your branch

This is not necessary (and in fact, confuses the diff). This is sometimes useful for large changes to expose new test failures due to changes in master, but this is not applicable here.

**Comment by Fromoriss** [comment]
Okay, now it's already existing, but last commits make it possible to order with a plushie crate and meta.json is updated with note, that moth plushie from /tg/

**Comment by lzk228** [comment]
@Fromoriss you added commits to your branch, that already was merged to wizard/master, so that because diff has "new" moth plushie, you should merge master in your branch now

File: Resources/Textures/Objects/Fun/toys.rsi/meta.json [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432, orb, orb-inhand-left and orb-inhand-right created by Pancake, plushie_diona and plushie_diona1 created by discord user Deos#5630, toy-mouse-equipped-HELMET is a resprited 1-equipped-HELMET in mouse.rsi by PuroSlavKing (Github), plushie_xeno by LinkUyx#6557, plushie_hampter by RenLou#4333, beachball taken from https://github.com/ss220-space/Paradise/commit/662c08272acd7be79531550919f56f846726eabb, beachb-inhand by ;3#1161",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432, orb, orb-inhand-left and orb-inhand-right created by Pancake, plushie_diona and plushie_diona1 created by discord user Deos#5630, toy-mouse-equipped-HELMET is a resprited 1-equipped-HELMET in mouse.rsi by PuroSlavKing (Github), plushie_xeno by LinkUyx#6557, plushie_hampter by RenLou#4333, beachball taken from https://github.com/ss220-space/Paradise/commit/662c08272acd7be79531550919f56f846726eabb, beachb-inhand by ;3#1161", moth plushie taken from tgstation by Fromoriss,
```
Comment (lzk228) [inline Resources/Textures/Objects/Fun/toys.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432, orb, orb-inhand-left and orb-inhand-right created by Pancake, plushie_diona and plushie_diona1 created by discord user Deos#5630, toy-mouse-equipped-HELMET is a resprited 1-equipped-HELMET in mouse.rsi by PuroSlavKing (Github), plushie_xeno by LinkUyx#6557, plushie_hampter by RenLou#4333, beachball taken from https://github.com/ss220-space/Paradise/commit/662c08272acd7be79531550919f56f846726eabb, beachb-inhand by ;3#1161, moth plushie taken from tgstation by Fromoriss",
```

**Comment by UbaserB** [comment]
why is it deleted this makes me so sad

**Comment by Just-a-Unity-Dev** [comment]
closed for some reason, im assuming they accidentally deleted the branch

**Comment by lzk228** [comment]
> why is it deleted this makes me so sad

#19828

**Comment by Just-a-Unity-Dev** [comment]
forgetf

**Pull request closed without merging**